app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    fetch_user_by_email
+    if @user&.authenticate params[:session][:password]
+      log_in @user
+      redirect_to admin_root_url if @user.role == 1
+      redirect_to root_url if @user.role == 2
```

Comment:
hehe, role tìm hiểu enum in rails nhé
`@user.admin?` thay vì `@user.role == 1` 

vào line 9 ko cần if nữa nhé, nghiễm nhiên để nó vào trang root cho khách hàng luôn chứ

Reply:
da ok